lib/ios-deploy.js

```diff
@@ -28,30 +30,37 @@ class IOSDeploy {
   }
 
   async install (app) {
+    const start = process.hrtime();
     try {
       const bundlePathOnPhone = await this.pushAppBundle(app);
-      await this.installApplcation(bundlePathOnPhone);
+      await this.installApplication(bundlePathOnPhone);
+      log.info(`Installation is successful after ${process.hrtime(start)[0]}s`);
```

Comment:
this can take less than one second. I'd rather consider nanoseconds as well. See https://github.com/appium/appium-base-driver/blob/43f4d2947390847a21ee16e506846fc0ed0f398e/lib/basedriver/helpers.js#L229

Reply:
Gotta be honest, new Date looks nicer since it gives ms out of the box. I will fallback to new Date